CMakeLists.txt

```diff
@@ -97,20 +97,9 @@ else()
   message(STATUS "Using eigen_catkin's Eigen in ${EIGEN_CATKIN_EIGEN_INCLUDE}")
 endif()
 
-cs_add_library(${PROJECT_NAME} src/dependency_tracker.cc)
-
-if(NOT USE_SYSTEM_EIGEN)
-  add_dependencies(${PROJECT_NAME} eigen_src)
```

Comment:
I think this is only to make sure that the library `${PROJECT_NAME}` depends on/is built after `eigen_src`. I removed the library (which only contained the empty `dependency_tracker.cc`) because I think it's not necessary.